avalon/tools/creator/app.py

```diff
@@ -324,6 +321,8 @@ def on_create(self):
             self.echo("Program error: %s" % str(e))
             raise
 
+        self.echo("Created %s .." % subset_name)
```

Comment:
In the Creator UI show that the creation has actually happened - in some applications this can be hard to notice, e.g. in Houdini you might not be in the context where you will see the creation happen and will be left wondering whether the UI actually did anything.